### ISSUE TITLE
checkPlpInstallation error with 'populationSettings'

### ISSUE THREAD

**Issue by Matyas123** [OHDSI/PatientLevelPrediction#304]
When checking installation with script chceckInstallation I got error for each test that 'populationSettings' does not found. When I changed 'populationSettings' for 'popSettings' in the function it is ok. 

**Set up (please run in R "sessionInfo()" and copy the output here):**
`> sessionInfo()
R version 4.2.1 (2022-06-23 ucrt)
Platform: x86_64-w64-mingw32/x64 (64-bit)
Running under: Windows 10 x64 (build 19043)

Matrix products: default

locale:
[1] LC_COLLATE=Czech_Czechia.utf8  LC_CTYPE=Czech_Czechia.utf8    LC_MONETARY=Czech_Czechia.utf8
[4] LC_NUMERIC=C                   LC_TIME=Czech_Czechia.utf8    

attached base packages:
[1] stats     graphics  grDevices utils     datasets  methods   base     

other attached packages:
[1] keras_2.9.0.9000             PatientLevelPrediction_5.0.5 FeatureExtraction_3.2.0      Andromeda_0.6.1             
[5] dplyr_1.0.9                  DatabaseConnector_5.0.4     

loaded via a namespace (and not attached):
  [1] TH.data_1.1-1        colorspace_2.0-3     deldir_1.0-6         ellipsis_0.3.2       htmlTable_2.4.1     
  [6] base64enc_0.1-3      fs_1.5.2             rstudioapi_0.13      MatrixModels_0.5-0   remotes_2.4.2       
 [11] bit64_4.0.5          mvtnorm_1.1-3        fansi_1.0.3          codetools_0.2-18     splines_4.2.1       
 [16] PRROC_1.3.1          memuse_4.2-1         cachem_1.0.6         knitr_1.39           pkgload_1.3.0       
 [21] zeallot_0.1.0        Formula_1.2-4        jsonlite_1.8.0       pROC_1.18.0          rJava_1.0-6         
 [26] cluster_2.1.3        SqlRender_1.9.2      dbplyr_2.2.1         png_0.1-7            tfruns_1.5.0        
 [31] shiny_1.7.2          compiler_4.2.1       httr_1.4.3           backports_1.4.1      assertthat_0.2.1    
 [36] Matrix_1.4-1         fastmap_1.1.0        cli_3.3.0            later_1.3.0          htmltools_0.5.3     
 [41] quantreg_5.94        prettyunits_1.1.1    tools_4.2.1          gtable_0.3.0         glue_1.6.2          
 [46] Rcpp_1.0.9           vctrs_0.4.1          nlme_3.1-157         xfun_0.32            stringr_1.4.0       
 [51] ps_1.7.1             mime_0.12            miniUI_0.1.1.1       lifecycle_1.0.1      devtools_2.4.4      
 [56] polspline_1.1.20     MASS_7.3-57          zoo_1.8-10           scales_1.2.0         hms_1.1.1           
 [61] promises_1.2.0.1     sandwich_3.0-2       SparseM_1.81         RColorBrewer_1.1-3   curl_4.3.2          
 [66] memoise_2.0.1        reticulate_1.25      gridExtra_2.3        ggplot2_3.3.6        rms_6.3-0           
 [71] rpart_4.1.16         latticeExtra_0.6-30  stringi_1.7.8        RSQLite_2.2.15       tensorflow_2.9.0    
 [76] checkmate_2.1.0      pkgbuild_1.3.1       rlang_1.0.4          pkgconfig_2.0.3      Cyclops_3.2.1       
 [81] lattice_0.20-45      purrr_0.3.4          ParallelLogger_3.0.1 htmlwidgets_1.5.4    bit_4.0.4           
 [86] processx_3.7.0       tidyselect_1.1.2     plyr_1.8.7           magrittr_2.0.3       R6_2.5.1            
 [91] generics_0.1.3       Hmisc_4.7-1          profvis_0.3.7        multcomp_1.4-20      DBI_1.1.3           
 [96] pillar_1.8.0         whisker_0.4          foreign_0.8-82       survival_3.3-1       nnet_7.3-17         
[101] tibble_3.1.8         crayon_1.5.1         interp_1.1-3         utf8_1.2.2           urlchecker_1.0.1    
[106] jpeg_0.1-9           usethis_2.1.6        grid_4.2.1           data.table_1.14.2    blob_1.2.3          
[111] callr_3.7.1          digest_0.6.29        xtable_1.8-4         tidyr_1.2.0          httpuv_1.6.5        
[116] munsell_0.5.0        sessioninfo_1.2.2   
>   checkPlpInstallation(connectionDetails = connectionDetails, python = T)
Checking database connectivity
Connecting using PostgreSQL driver
- Done

Checking R population
Generating covariates
Generating cohorts
Generating outcomes
Outcome is 0 or 1
- Done

Checking Models
Use timeStamp: TRUE
Error in runPlp(plpData = plpData, outcomeId = 2, analysisId = "test1", : object 'populationSettings' not found

- Ok
Use timeStamp: TRUE
Error in runPlp(plpData = plpData, outcomeId = 2, analysisId = "test2", : object 'populationSettings' not found

Use timeStamp: TRUE
Error in runPlp(plpData = plpData, outcomeId = 2, analysisId = "test3", : object 'populationSettings' not found

Use timeStamp: TRUE
Error in runPlp(plpData = plpData, outcomeId = 2, analysisId = "test4", : object 'populationSettings' not found

Use timeStamp: TRUE
Error in runPlp(plpData = plpData, outcomeId = 2, analysisId = "test5", : object 'populationSettings' not found

Use timeStamp: TRUE
Error in runPlp(plpData = plpData, outcomeId = 2, analysisId = "test6", : object 'populationSettings' not found

Use timeStamp: TRUE
Error in runPlp(plpData = plpData, outcomeId = 2, analysisId = "test7", : object 'populationSettings' not found

Use timeStamp: TRUE
Error in runPlp(plpData = plpData, outcomeId = 2, analysisId = "test8", : object 'populationSettings' not found

- Done

Checking support for large data objects
- Done

PatientLevelPrediction installation check completed...

Response code: 530052723915
Issue with database connection - did not connect
Issue with database connection - did not disconnect
Issue with lasso logistic regression
Issue with random forest
Issue with mlp
Issue with ada boost
Issue with decison tree
Issue with naive bayes
Issue with knn
Issue with gradient boosting machine
NULL`

**Comment by jreps** [comment]
Closing this as it is a duplicate of https://github.com/OHDSI/PatientLevelPrediction/issues/290
